tests/e2e/sample-applications/mysql-persistent/mysql-backup.yaml

```diff
@@ -6,5 +6,8 @@ metadata:
 spec:
   includedNamespaces:
   - mysql-persistent
+  # snapshotMoveData: true
+  # What is the goal of this file? Is it documented?
+  # Feel it is just trouble, would delete it (along with restore). anyone against?
```

Comment:
I thought I had to update this file for the E2E tests, so I see no meaning for it

Anyone thinks it is good to let it in the repo?

Reply:
right, going to revert this change